### PR TITLE
Clarifying reselection of SSN for taxes

### PR DESCRIPTION
**Pull request by SylLutze**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 1]
```diff
@@ -2531,6 +2531,17 @@ then fill in a new randomly generated SSN. This is designed to reflect
 undocumented immigrants who might use fake or no longer 
 valid SSNs to obtain employment. 
 
+Please reselect or regenerate a new SSN for each observation. 
+If a simulant has had multiple jobs within the prior year, please 
+reselect or regenerate a SSN for each job. This can overlap with 
+the prior selection (for example, if there is only one person in the 
+house to select an SSN from) but should be randomly rechoosen each
```
Comment (zmbc) [inline docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2538]:
```suggestion
house to select an SSN from) but should be randomly rechosen each 
```

**Review by zmbc — APPROVED** [review]

**Review by rmudambi — APPROVED** [review]

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 2]
```diff
@@ -2531,6 +2531,17 @@ then fill in a new randomly generated SSN. This is designed to reflect
...
 
+Please reselect or regenerate a new SSN for each observation. 
+If a simulant has had multiple jobs within the prior year, please 
+reselect or regenerate a SSN for each job. This can overlap with 
+the prior selection (for example, if there is only one person in the 
+house to select an SSN from) but should be randomly rechosen each 
+time. 
+
+While this likely adds more variation than should be expected to the 
+SSN's in the model, we think it is better to have more variation than
```
Comment (mattkappel) [inline docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2542]:
```suggestion
SSNs in the model, we think it is better to have more variation than 
```

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 3]
```diff
@@ -2531,6 +2531,17 @@ then fill in a new randomly generated SSN. This is designed to reflect
 undocumented immigrants who might use fake or no longer 
 valid SSNs to obtain employment. 
 
+Please reselect or regenerate a new SSN for each observation. 
+If a simulant has had multiple jobs within the prior year, please 
+reselect or regenerate a SSN for each job. This can overlap with 
+the prior selection (for example, if there is only one person in the 
+house to select an SSN from) but should be randomly rechosen each 
+time. 
+
```
[the remark this answers is not in the document]
Comment (zmbc) [reply docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2540]: This is referring to tax year/calendar year, so 2024 if the year is 2025.

Workers can't have multiple jobs _on a time step_, but they can have multiple jobs on different time steps within the same year. This leads to them having multiple W2s for that tax year.

[the remark this answers is not in the document]
Comment (SylLutze) [reply docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2540]: Zeb is correct here, but I also added some more clarification to the documents.

**Pull request merged**